build/agent/Dockerfile

```diff
@@ -11,7 +11,7 @@ ENV ENTRYPOINT_CMD="/entrypoint.sh"
 
 RUN cd cmd/agent;go build -o /runner -mod mod -a .
 
-FROM justb4/jmeter
+FROM justb4/jmeter:5.5
```

Comment:
Isn't it better to use multiple versions of JMeter like we have for Cypress, if we use latest, it might update at some point to 5.6 and start to fail for users because they were expecting version 5.5.

JMeter is very fragile between versions